app/views/admin/tool_rentings/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t('tool_renting.title')) %>
+<div class="col-md-8 col-md-offset-2 col-sm-12 reg-page">
+  <div class="headline">
+    <h2><%= t('tool_renting.edit') %></h2>
```

Comment:
No, the `title` method will now set the title for the web tab and print the text.

So remove the first line completely and just wrap like:
`<h2><%= title(t('tool_renting.edit')) %></h2>`